### ISSUE TITLE
Boot stuck on "VTloading....."

### ISSUE THREAD

**Issue by TeoColuccio** [ventoy/Ventoy#1374]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.64

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### BIOS Mode

Legacy BIOS Mode

### Partition Style

MBR

### Disk Capacity

32GB

### Disk Manufacturer

Kingston

### What happened?

I am trying to use a usb stick with ventoy on a hp pavilion dv6 pc. This I have a legacy bios, so ventoy was written simply with Ventoy2Disk.sh -i / dev / sdc.
Now the problem is that when I go to boot from usb, I find myself a black screen stuck indefinitely with only written "VTloading .....".
I have tried various iso images in multiple different formats. I first had fat32 left, then I tried NTFS and finally ext4, but nothing has changed.
Finally I also tried to reformat with the secure boot -s option (even if this is not necessary for my bios), but this too was useless. What else can I try?

**Comment by crasadure** [comment]
You can try text mode
Pls,read this page
https://forums.ventoy.net/showthread.php?tid=1510

**Comment by ventoy** [comment]
Some old legacy bios machine boot the USB device fine only if the 1st partition of the USB device start from sector 63.
Ventoy's 1st partition is start from sector 2048. So if your machine has such requirement then it can not use Ventoy.

**Comment by TeoColuccio** [comment]
> You can try text mode Pls,read this page https://forums.ventoy.net/showthread.php?tid=1510

Nothing to do. I set the cli mode through VentoyPlugson.sh, and in fact on other PCs I see the change, but the hp pavilion always remains blocked on VTloading.

**Comment by TeoColuccio** [comment]
> Some old legacy bios machine boot the USB device fine only if the 1st partition of the USB device start from sector 63. Ventoy's 1st partition is start from sector 2048. So if your machine has such requirement then it can not use Ventoy.

Thanks for the reply, there is nothing I can attempt then?

**Comment by steve6375** [comment]
Try other PCs/Notebooks?

**Comment by TeoColuccio** [comment]
> Try other PCs/Notebook

As I said, on other notebooks, ventoy works great in both gui and cli modes. It is only on the hp pavilion DV6 that, regardless of gui or cli mode, it gets stuck on VTloading.....?

**Comment by steve6375** [comment]
Sorry, I missed that!
So maybe try another boot manager - e.g. Easy2Boot which can also load Ventoy (you need a Windows 10 system to prepare the USB drive). E2B boots to grub4dos but then loads agFM or Ventoy which are both grub2 based.
so see if any issues legacy booting to E2B menu (grub4dos), agFM menu (grub2) and Ventoy menu (grub2) systems.

**Comment by lickumap** [comment]
请问  之前进得去 现在死活进不去  百度到几个答案  说是内存太大  之前4+4  现在4+4+2+2   



台式机没问题 笔记本死活不行    重做也不行   

答案2  可以bios关闭安全启动，老点的电脑建议插USB2.0上。   

没有安全这个选项   USB四个口都试过了  

最近在学怎么建站 用PHPstudy  的 然后用工具改了IP啥的 然后二天开机就连不到网 转圈很久

**Comment by drws** [comment]
I'm confirming this is still the case with `v1.0.74` and th eHP dv6.

**Comment by steve6375** [comment]
Not everyone has a HP Pavilion dv6 pc on there desk - so we cannot test it to  find out what could be the cause.
We would need information as I requested previously on Jan 13.
We would like to know answers (also a YUMI USB drive works or if a Rufus USB drive works if prepared with a Linux ISO or a Windows 10 ISO).

**Comment by sonic011gamer** [comment]
This might not be the case for this issue, but for me on a legacy BIOS system, disabling "Secure Boot support" in Ventoy fixed it.

**Comment by Pitam-Poudel** [comment]
same issue on hp pavilion dv7

**Comment by sonic011gamer** [comment]
> @sonic011gamer: Disabling secure boot should have no effect on legacy booting?

It _shouldn't_, but it did solve the issue on a HP Pavilion G6.

**Comment by steve6375** [comment]
Some BIOSes do not allow legacy boot if Secure Boot is enabled in the BIOS.
Are you sure it is legacy booting when Secure Boot is enabled in the BIOS? Try renaming the \EFI partition on Partition 2 of the Ventoy USB drive to \EFIxx. Then see if you get the same symptoms with Secure Boot enabled.

**Comment by Pitam-Poudel** [comment]
My laptop doesn't have secure boot option. I have hp pavilion dv7 laptop . I tried Ventoy with different configurtions too many times but is still not working for me.

**Comment by nexus-arch** [comment]
For me worked change option from bios boot to uefi boot

**Comment by vp-endava** [comment]
Hello I have a HP..  DV6 with an I7, and previous version of ventoy worked, but the last one not. It stuck on "VTloading..." no EFI thing on boot options

**Comment by veplechuc** [comment]
Hi, I have  a HP DV6 with an I7 and have the same issue on "VTloading..." stuck, on previous version it worked, but now I does not. Any workaround on this?

**Comment by super302** [comment]
我遇到了同样的问题，开机卡在“VTloading”，最后是换了个内存条后恢复正常的。

**Comment by Blueshawk** [comment]
> 
I'm also trying to get around this, but only have the machine while I work on it.
 Renaming the EFI dir EFIxx didn't work on the Hp pavilion dv7.
 Easy2boot does boot to menus on this machine, but is a mess. I got it as far as attempting to boot Mint21.1 by specifying missing bootloader, but that failed to boot also.
 Fwiw, a clean mint usb drive image (dd) also fails to boot to grub on this machine.

**Comment by NintendoManiac64** [comment]
I've also recently come into possession of an HP dv7 (specifically the dv7t-6100) with this exact same issue and have to work around it using the old fashioned "Rufus" method of installing ISOs.

The one thing I can add to this discussion however is that Ventoy *does* work when connected via the internal SATA.

Also, curiously, I'm in possession of an HP DM1 laptop from around the same year, albeit with an AMD CPU rather than an Intel one (which the dv7 uses), yet the DM1 has no issues with Ventoy...  I've also been able to use Ventoy on both much older devices (e.g. from laptops from 2006 and 2007) as well as much newer devices (gaming handhelds with Ryzen 4000 and Ryzen 5000, so hardware that isn't even 5 years old)

**Comment by beastlykings** [comment]
Another dv7 user here, stuck with the same issue. It's not a big deal, I suppose. I only wanted to use this for testing purposes, playing with Ventoy for the first time. It's just a bummer that it still doesn't work.

I have version 1.0.94.

Thanks all!

**Comment by mbnoimi** [comment]
I tried another method for creating Ventoy pen by restoring `ventoy-1.0.95-livecd.iso` into my USB pen as following:

Restore Disk Image...
![image](https://github.com/ventoy/Ventoy/assets/284547/ed26c404-2a65-4cc5-bf4c-3c950311891c)

Unfortunately I got a similar message:
`GRLoading...`

**Comment by mbnoimi** [comment]
**IMPORTANT:** I could run same USB pen on my friend laptop without any problem. This issue occurs on my laptop only :(

My BIOS configurations:
![IMG_20230829_144953](https://github.com/ventoy/Ventoy/assets/284547/d9ed6c70-07ba-4b75-bfa5-fc06ea356cbb)
![IMG_20230829_145035](https://github.com/ventoy/Ventoy/assets/284547/e2f4e908-5d9d-468c-89af-441386b7221e)

**Comment by NintendoManiac64** [comment]
> I configured the BIOS to legacy boot.
Is there a particular reason for this? I personally find it better to install the OS itself via UEFI and then add a secondary legacy bootloader so that your hard drive/SSD can be booted on both types.

More info:
- https://forums.linuxmint.com/viewtopic.php?f=46&t=375211

> BTW, Ventoy was working fine using the same USB pen and laptop But after updating Ventoy to recent version (1.0.95) I no longer able to making it work!

Then what happens if you downgrade to an older version? All (or at least most?) of the old versions of Ventoy are still available for download.

If the version you were using previously was substantially old, then the trick is to chose a version that's 64 revisions back, that being v1.0.31. If that works, then go half-way newer to 32 revisions back, that being v1.0.63. If it works or doesn't work, then try 16 revisions newer or older, and continue doing this "cutting in half" thing until you land on the exact revision where it last worked.

> I really need to re-install Linux Mint 21.2 into my laptop
> I could run same USB pen on my friend laptop without any problem

You could try things the old-fashioned way: writing the Linux Mint 21.2 ISO using Mint's built-in disk image writer and just overwrite the entire USB disk with just that ISO temporarily.

Another derpy workaround would be, If both your laptop and your friend's laptop do *NOT* have Nvidia graphics, then you can always just put your internal hard drive/SSD into your friend's laptop, install Mint 21.2 onto your hard drive/SSD, and then put your hard drive/SSD back into your laptop (this works with any combination of AMD and Intel CPUs and graphics—just no Nvidia).

**Comment by mbnoimi** [comment]
Thank you @NintendoManiac64 
I figured out what's wrong. Some guy on Reddit advice me to unplug all USB devices and keep only USB pen. I discovered that my wireless mouse conflicts with USB pen on boot only. Unplugged wireless mouse on boot then everything went fine,

**Comment by tech128** [comment]
Well as luck would have it, I received one of these laptops from a customer who upgraded.  It seems to have the same issue of not loading any version of ventoy.  I tried to load 1.0.95 from an easy2boot drive and that freezes as well.  So I don't believe it falls into the realm of the sector 63 issue.

I do have a workaround method that works well.  If I run duet efi bootloader and then run ventoy from there, it works fine.

It would be nice to not have to do that though.

Just to add, I cannot run grubfm at all either.  It freezes when I try to run it as well.

If you need me to run debug builds let me know.

**Comment by breversa** [comment]
@tech128 What exact laptop do you have?

I recently picked up a 2011 HP Notebook dv6 (will provide exact model later) that is BIOS-only (v1B if that helps) and gets stuck on `VTloading…` as well.

By chance, I was able to boot and install Linux Mint using the optical drive, but I’d like to update the BIOS to v1B **rev. A** using Hiren’s BootCD (PE).

**Comment by breversa** [comment]
@tech128 What exact laptop do you have?

I recently picked up a 2011 HP Notebook dv6 (will provide exact model later) that is BIOS-only (v1B if that helps) and gets stuck on `VTloading…` as well with the current latest version v1.0.97.

By chance, I was able to boot and install Linux Mint using the optical drive, but I’d like to update the BIOS to v1B **rev. A** using Hiren’s BootCD (PE).

**Comment by NintendoManiac64** [comment]
> I’d like to update the BIOS to v1B **rev. A** using Hiren’s BootCD (PE).

The trick is to "install" Hiren's BootCD PE via the traditional method of writing an ISO to a USB drive, that being Rufus, Etcher, etc.

...and actually, since you said you used the optical drive, you could just as easily do that for Hiren's BootCD PE since it comes in ISO format and is roughly the same file size as Linux Mint.

**Comment by tech128** [comment]
> @tech128 What exact laptop do you have?
> 
> I recently picked up a 2011 HP Pavilion dv6-6057ef that is BIOS-only (v1B if that helps) and gets stuck on `VTloading…` as well with the current latest version v1.0.97.

Just checked mine right now.  It is a HP Pavilion dv6-6150us.  BIOS Revision: 15.27

**Comment by breversa** [comment]
@NintendoManiac64 Thanks for trying to help. I did not need your tips eventually, but maybe they’ll help someone else.

@tech128 Maybe we could start a "Does not work on"-list?

**Comment by tech128** [comment]
> @tech128 Maybe we could start a "Does not work on"-list?

Yeah that'd be fine.  But in my job I've come across less older systems than I used to.  So it might take awhile to find these systems that ventoy doesn't want to work on.

I'm more interested in finding the cause of the issues.  There has to be some reason it gets stuck.

When I get some time, I'd like to try to see if modifying the boot image, so the first partition starts at sector 63 and also disable secure boot support in ventoy and see how that goes.

**Comment by NintendoManiac64** [comment]
>  But in my job I've come across less older systems than I used to.

It has nothing to do with it being an older system as my own HP laptop this occurs on is in fact the second *newest* laptop I own (my newest, a Thinkpad T420, is only half a year newer), yet I've had no issues with Ventoy on the various older laptops (at least as far back as DDR2-era system) including another HP that's only half a year older but uses AMD instead of Intel.

**Comment by breversa** [comment]
> So did you try disconnecting all usb devices except the Ventoy usb drive?

I have.
However, I have since returned that laptop to their owner, so I'm no longer able to test anything. Thanks for the help, though. I hope other people will be able to contribute.

**Comment by enygmate** [comment]
Also experiencing this issue, on an HP Pavilion DV7.
Sucks cuz that's all I got to troubleshoot issues on the computer right now. As if this computer wasn't shitty enough, it doesn't even support Ventoy, great.
😀👍

**Comment by steve6375** [comment]
Easy2boot now has Ventoy 1.0.97 so it may be worth trying. Male a new e2b usb drive with agFM and Ventoy added when prompted .
When legacy booting you first boot to grub4dos e2b menu and then boot to Ventoy for Easy2boot from the e2b menu.
I also suggest loading the grub4dos usb driver from the e2b utilities menu first before loading Ventoy and see if that loads Isos better, as it's the usb side of things that seems to be the bioses problem as booting directly to Ventoy from internal sata hard disk seems to work.

**Comment by tech128** [comment]
Steve,

I tried usin ventoy 1.0.97 right when it first came out hopin it would fix the issue.  No dice.  Also just tried to use grub4dos usb driver from easy2boot, it always says no usb device detected.  I also tried plop boot manager 6 in the hopes that a different usb stack would work.  That hangs on tryin to detect ehci for some odd reason.  Also another weird thing I noticed.  In addition to grubfm not wantin to load, the system locks anytime i try to load an initrd after a kernel memdisk.

**Comment by TeoColuccio** [comment]
I am sure there is some incompatibility problem with the latest versions of grub. 
Because after trying unsuccessfully to use the latest Ubuntu ISO (24.04) in the latest version of ventoy, I flashed this directly using just dd and it still didn't work. 
Flashing an older Ubuntu iso instead (I tried 19.10), again using dd, this one booted without any problems.
So it is a problem of the bootloader, i.e., grub.

**Comment by TeoColuccio** [comment]
I don't know if this other information is also helpful.. the problem evidently is ALSO the bios.
Because I just tried burning the same iso of Ubuntu 24.04 to dvd and booting from internal dvd drive, it boots regularly.

**Comment by lucianm** [comment]
> I am sure there is some incompatibility problem with the latest versions of grub. Because after trying unsuccessfully to use the latest Ubuntu ISO (24.04) in the latest version of ventoy, I flashed this directly using just dd and it still didn't work. Flashing an older Ubuntu iso instead (I tried 19.10), again using dd, this one booted without any problems. So it is a problem of the bootloader, i.e., grub.

Does "VTLoading....." come from Grub? Even on a freshly "ventoyed" USB medium?

**Comment by Pldm24** [comment]
Hello, I have HP Pavilion dv6 and I have the same problem with ventoy. Maybe I can help with something?

**Comment by Pldm24** [comment]
Hello, I have HP Pavilion dv6 and I have the same problem with ventoy. Can I help with it? (Sorry if you saw the same message twice)

**Comment by TeoColuccio** [comment]
> > I am sure there is some incompatibility problem with the latest versions of grub. Because after trying unsuccessfully to use the latest Ubuntu ISO (24.04) in the latest version of ventoy, I flashed this directly using just dd and it still didn't work. Flashing an older Ubuntu iso instead (I tried 19.10), again using dd, this one booted without any problems. So it is a problem of the bootloader, i.e., grub.
> 
> Does "VTLoading....." come from Grub? Even on a freshly "ventoyed" USB medium?

Yes, I think it comes out of the version of GRUB embedded in ventoy.

**Comment by Pa121-hub** [comment]
Same issue  with HP Pavilion dv6

**Comment by beastlykings** [comment]
Just bumping this, as I've just tested 1.1.05 on my dv7 and I'm still having the same issue, "VTloading......"

It's a bummer because I was gearing up to rapidly test several Linux distros in a row tonight, but I'm stuck because I don't want to have to flash each one.

**Comment by albiniscat** [comment]
I've had the same issue where the BIOS sequence stalls at the "VTLoading......." screen - on an older HP Pavilion dv6 (from 2011-ish).

Tried a number of things, including YUMI exFat (which also installs Ventoy resulting in similar problems).
Was able to boot Mint using Rufus (DD setting rather than ISO), but no luck with Ventoy, despite a lot of experimentation.

I've had better success with Easy2Boot (version 2.21).

When you install Easy2Boot on a USB stick, the CLI (command line interface) window will ask you two questions:

- Do you want to add agFM boot files?
- Do you want to add Ventoy?

I selected **Yes** for the agFM install and **No** for the Ventoy install.

(E2B will still create a Ventoy folder on the USB drive).

You can then copy the Linux .ISO files to the _ISO/LINUX folder.

Restarting the PC then loads into E2B from the USB.

It loads correctly:

**Linux Mint   (LinuxMint-22.2-cinnamon-64bit)
POP!_OS   (pop-os-22.04_amd64_intel_58)**

However not for:

Fedora   (Fedora-KDE-Desktop_Live-42-1.1.x86_64)   -- doesn't progress beyond the "Welcome to GRUB!" message (top left of screen)
Ubuntu   (Ubuntu-24.04.3-desktop-amd64)   -- blank screen, doesn't progress


The last two could be due to the compatibility of the OS versions with the older PC.

I'll investigate this further and report back here.

Above, incase any is useful to someone finding this thread.  :-)



### Specs:
HP Pavilion dv6 6156ea
i5-2410M (64-bit processor)

64Gb SanDisk Ultra USB (3.0).  For boot attempts, inserted into USB 2.0 port and took out mouse USB Bluetooth receiver.

BIOS HP F.1B (23/10/22)  SMBIOS v2.7
Non UEFI, Legacy BIOS only.  Secure Boot not on this BIOS.

**Comment by testudo92** [comment]
For those using an HP Pavilion DV series - After years of futility getting a USB to boot, I finally had success booting from a USB with an Ubuntu image created using UNETBOOTIN on my Pavilion DV7.  It installs a SYSLINUX boot loader that plays nice with legacy BIOS like the HP DV series. Hope that helps some of you.

**Comment by testudo92** [comment]
Correct program name is call UNETBOOTIN.
